components/Tab/src/Tab.tsx

```diff
@@ -1,63 +1,25 @@
-import React, { ReactElement } from 'react';
-import { Tab as MaterialTab } from '@material-ui/core';
-import BaseProps from '@gemeente-denhaag/baseprops';
+import React, { HTMLAttributes, forwardRef } from 'react';
 import './styles/tab.scss';
 import clsx from 'clsx';
 
-export interface TabProps extends Omit<BaseProps, 'classes'> {
-  /**
-   * If true, the tab will be disabled.
-   */
-  disabled?: boolean;
-
-  /**
-   * If true, the keyboard focus ripple will be disabled.
-   */
-  disableFocusRipple?: boolean;
-
-  /**
-   * If true, the ripple effect will be disabled.
-   */
-  disableRipple?: boolean;
-
-  /**
-   * The icon element.
-   */
-  icon?: ReactElement;
-
-  /**
-   * The label element.
-   */
-  label: React.ReactNode;
-
-  /**
-   * You can provide your own value.
-   * Otherwise, we fallback to the child position index.
-   */
-  value?: number | string;
-
-  /**
-   * Tab labels appear in a single row.
-   * They can use a second line if needed.
-   */
-  wrapped?: boolean;
+export interface TabProps extends HTMLAttributes<HTMLDivElement> {
+  selected?: boolean;
 }
 
 /**
  * Tabs make it easy to explore and switch between different views.
  * @param props The properties of a Tab component.
  * @constructor Construct an instance of Tab.
  */
-export const Tab: React.FC<TabProps> = (props: TabProps) => {
-  const rootClassNames = clsx('denhaag-tabs__tab', props.className);
+export const Tab = forwardRef<HTMLDivElement, TabProps>(({ selected, className, ...props }, ref) => {
+  const rootClassNames = clsx('denhaag-tabs__tab', selected ? 'denhaag-tabs__tab--selected' : '', className);
   return (
-    <MaterialTab
-      {...props}
-      classes={{ selected: 'denhaag-tabs__tab--selected' }}
-      className={rootClassNames}
-      disableRipple
-    />
+    <div {...props} ref={ref} role="tab" aria-selected={selected} className={rootClassNames}>
+      {props.children}
+    </div>
   );
-};
+});
+
+Tab.displayName = 'Tab';
```

Comment:
Do we need this extra displayName if we are using a const?

Reply:
@Borai It's necessary to prevent this eslint error:
`denhaag\components\Tab\src\Tab.tsx
  14:20  error  Component definition is missing display name  react/display-name`